content/docs/next-release/frontend-ui.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Frontend/UI
+weight: 10
+---
+
+Jaeger Web UI is implemented in Javascript using popular open source frameworks like React. Several performance improvements have been released in v1.0 to allow the UI to efficiently deal with large volumes of data, and to display traces with tens of thousands of spans (e.g. we tried a trace with 80,000 spans).
+
+## Embed Mode
+
+Enhancement to embed mode in Jaeger UI for some components allowing us to integrate it into other applications.
```

Comment:
I suggest rephrasing this paragraph to:

-----

>Starting with version **[TODO]**, Jaeger UI provides an "embedded" layout mode which is intended to support integrating Jaeger UI into other applications. Currently (as of `v0`), the approach taken is to remove various UI elements from the page to make the UI better suited for space-constrained layouts.
>
>The embedded mode is induced and configured via URL query parameters.
>
>To enter embedded mode, the `uiEmbed=v0` query parameter and value must be added to the URL. For example, the following URL will show the trace with ID `abc123` in embedded mode:
>
>```
>https://example.com/trace/abc123?uiEmbed=v0
>```
>
>`uiEmbed=v0` is required.
>
>Further, each page supported has an <img src="/static/img/frontend-ui/embed-open-icon.png" style="width: 20px; height:20px;" alt="Embed open window"> button added that will open the non-embedded page in a new tab.

-----
What do you think? 

Here is the markdown:

    Starting with version **[TODO]**, Jaeger UI provides an "embedded" layout mode which is intended to support integrating Jaeger UI into other applications. Currently (as of `v0`), the approach taken is to remove various UI elements from the page to make the UI better suited for space-constrained layouts.

    The embedded mode is induced and configured via URL query parameters.

    To enter embedded mode, the `uiEmbed=v0` query parameter and value must be added to the URL. For example, the following URL will show the trace with ID `abc123` in embedded mode:

    ```
    https://example.com/trace/abc123?uiEmbed=v0
    ```

    `uiEmbed=v0` is required.
    
    Further, each page supported has an <img src="/static/img/frontend-ui/embed-open-icon.png" style="width: 20px; height:20px;" alt="Embed open window"> button added that will open the non-embedded page in a new tab.